package.json

```diff
@@ -4,8 +4,10 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "prop-types": "^15.6.2",
     "react": "16.6.0",
     "react-dom": "16.6.0",
+    "react-router-dom": "^4.3.1",
```

Comment:
Could you use exacts version? Without the ` ^`. In order to avoid potential version issues.
```
"prop-types": "15.6.2",
"react-router-dom": "4.3.1",
```